torch/csrc/dynamo/eval_frame.c

```diff
@@ -592,6 +592,12 @@ inline static PyObject* eval_custom_code(
   THP_EVAL_API_FRAME_OBJECT* shadow = shadow_obj->f_frame;
   Py_XINCREF(frame->f_func->func_closure);
   shadow->f_func->func_closure = frame->f_func->func_closure;
+  // NOTE: in Python 3.11.1+, PyFrame_New changes prev_instr, causing
```

Comment:
I'm not sure what this comment means? Dont we want the frame to be properly valid, including prev_instr ?
Why are we resetting the value to something invalid?

Reply:
It doesn't seem like CPython uses PyFrame_New for regular frame evaluation. PyFrame_New marks the frame as "complete" (by setting prev_instr to the first RESUME instruction), but frames for frame evaluation expect prev_instr to be the instruction before the first instruction. There may be more underlying issues with using PyFrame_New, but I'll address them in future PRs.

Reply:
Ok sounds good to unblock but most we should most likely clean it up.